prac_06/programming_language.py

```diff
@@ -0,0 +1,25 @@
+# Adrianto Hartono
+# 14347930
+
+"""
+Programming Languages
+
+Expect: 20 min
+Actually: 15 min
+"""
+
+class ProgrammingLanguage:
+    def __init__(self, name="", typing="", reflection="", year=""):
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def is_dynamic(self):
+        if self.typing.title() == "Dynamic":
+            return True
+        else:
+            return False
+
```

Comment:
Same with the previous file, you can just return self.typing.title() == "Dynamic" instead of using the if function